2025/build-tooling/slides.md

```diff
@@ -79,6 +77,12 @@ TODO: split into multiple slides or reduce. Move TypeScript out further down?
     - Replace CDN script tags with NPM imports
     - Link to map-components NPM get started
 
+---
+
+# Demo summary
+
+- Quick to get started
+- Live reload and hot module replacement -> fast feedback loop
```

Comment:
```suggestion
- Live reload and Hot Module Replacement -> fast feedback loop
```

technically Hot Module Replacement requires live reload, so we could mention just one or the other

Hot Module Replacement is more inclusive, but also less familiar term - we could say just live reload, but show during the demo how this live reload preserves state?
or call it "live update" / "realtime update" instead?
or should we not worry about using complex terms?

Reply:
I like "live update"